<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTC_main_widget</class>
 <widget class="QWidget" name="QTC_main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="settings_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>7</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="save_to_button">
       <property name="text">
        <string>Output directory</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="save_settings_button">
       <property name="text">
        <string>Save settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="proj_comboBox"/>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="filename"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="load_settings_button">
       <property name="text">
        <string>Load settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="sensor_label">
       <property name="text">
        <string>Sensor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="7">
      <widget class="QLineEdit" name="output_dir_edit"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="operator_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Operator</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="project_label">
       <property name="text">
        <string>Project</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QComboBox" name="operator_comboBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="sensor_comboBox"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="filename_label">
       <property name="text">
        <string>Output filename</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="PlotWidget" name="IV_plot">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>500</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QFrame" name="DoIVCVframe">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1" rowspan="4">
       <widget class="QFrame" name="IV_settings_settings_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="max_voltage_label_2">
           <property name="text">
            <string>Maximum bias voltage [V]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="max_voltage_2">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>-1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="complience_label_2">
           <property name="text">
            <string>Complience [uA]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="complience_2">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>-50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="voltages_steps_label_2">
           <property name="text">
            <string>Voltage steps</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="voltage_steps_2">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="doIV_checkBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Do IV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="refinement_settings_label_2">
        <property name="text">
         <string>Voltage setp refinement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="general_settings_label_2">
        <property name="text">
         <string>General IVCV parameters</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="doCV_checkBox_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Do CV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="doRef_checkBox_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Do refinement</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="QFrame" name="Refinement_settings_label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="minimu_refinement_label_2">
           <property name="text">
            <string>Minimum bias voltage [V]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="min_refine_spin_2">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maximum_refinement_label_2">
           <property name="text">
            <string>Maximum bias voltage [V]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="max_refine_spin_2">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>-50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="voltages_steps_refine_label_2">
           <property name="text">
            <string>Refinement voltage steps</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="refine_step_spin_2">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Maximum voltage CV [V]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="max_voltage_CV">
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>-400.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="PlotWidget" name="CV_plot">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <layout class="QGridLayout" name="controls_layout">
     <item row="0" column="3">
      <widget class="QLCDNumber" name="bias_voltage_lcd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="quit_button">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="current_pad_lcd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="stop_button">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../images/svg_resources.qrc">
         <normaloff>:/stop/stop.svg</normaloff>:/stop/stop.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="4" colspan="4">
      <widget class="QLabel" name="event_log">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>400</height>
        </size>
       </property>
       <property name="text">
        <string>No errors =)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="progress_label">
       <property name="text">
        <string>Progress:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="state_label">
       <property name="text">
        <string>Current State:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="bias_voltage_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Bias Voltage:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../images/svg_resources.qrc">
         <normaloff>:/play/start.svg</normaloff>:/play/start.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="state_indi">
       <property name="font">
        <font>
         <family>Perpetua Titling MT</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Idle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="event_log_label">
       <property name="text">
        <string>Event Log:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="current_pad_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Current pad:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="bad_pads_lcd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="bad_pads_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Bad pads:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="environement_control_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="7" column="6">
      <widget class="QDoubleSpinBox" name="min_hum_spin"/>
     </item>
     <item row="4" column="8" rowspan="4" colspan="2">
      <widget class="QFrame" name="lcd_display">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="temp_lcd_label">
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="temp_lcd">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 44, 16), stop:1 rgb(255, 85, 70));</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="hum_lcd_label">
          <property name="text">
           <string>Humidity</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="hum_lcd">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(21, 71, 255), stop:1 rgb(53, 56, 255));</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dew_point_lcd_label">
          <property name="text">
           <string>Dew point</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="dew_point_lcd">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(11, 208, 67), stop:1 rgb(25, 161, 48));</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QCheckBox" name="log_env_check">
       <property name="text">
        <string>Log environment</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QDoubleSpinBox" name="min_temp_spin"/>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="max_label">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QDoubleSpinBox" name="max_hum_spin"/>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="humm_levels_label">
       <property name="text">
        <string>Humidity levels</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QCheckBox" name="env_control_check">
       <property name="text">
        <string>Environment control</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLabel" name="min_hum">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="min_label">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QLabel" name="temp_levels_label">
       <property name="text">
        <string>Temperature levels</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QProgressBar" name="humidity_bar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QDoubleSpinBox" name="max_temp_spin"/>
     </item>
     <item row="0" column="4" rowspan="4" colspan="5">
      <widget class="PlotWidget" name="pyqtPlotItem">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="max_hum">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="max_temp">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="min_temp">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QProgressBar" name="temperature_bar">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/svg_resources.qrc"/>
 </resources>
 <connections/>
</ui>
